packages/liveblocks-core/src/protocol/ServerMsg.ts

```diff
@@ -223,6 +223,7 @@ export type YDocUpdateServerMsg = {
   readonly stateVector: string | null; // server's state vector, sent in response to fetch
   readonly guid?: string; // an optional guid to identify which subdoc this update to
   readonly v2?: boolean; // whether this is a v2 update
+  readonly remoteSnapshot: string; // The snapshot of server's document. Used to detect if the client has the latest version of the document.
```

Comment:
This is a new property in `ServerMsgCode.UPDATE_YDOC` type messages that the server sends on each update. A snapshot includes state vector and delete set, so in cases when this message already includes the state vector (initial sync step), we'll have duplication, but state vectors and delete sets are both minimal values, so it shouldn't affect the message size.

Reply:
I'm worried about how big this may get. Have you tested it with a large document?